modules/static/bufbuild/reflect/rsync.incl

```diff
@@ -0,0 +1,6 @@
++ /LICENSE
++ /README.md
++ /buf.yaml
```

Comment:
This one does include the `buf.yaml` because it's maintained by us. See https://github.com/bufbuild/modules/pull/344#discussion_r1404443184